crates/sui-light-client/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "sui-light-client"
+version = "0.0.0"
+authors = ["Mysten Labs <build@mystenlabs.com>"]
+license = "Apache-2.0"
+publish = false
+edition = "2021"
+
+[dependencies]
+anyhow.workspace = true
+async-trait.workspace = true
+axum.workspace = true
+bcs.workspace = true
+byteorder.workspace = true
+bytes.workspace = true
+chrono.workspace = true
+clap.workspace = true
+csv.workspace = true
+move-core-types.workspace = true
+object_store.workspace = true
+num_enum.workspace = true
+prometheus.workspace = true
+serde.workspace = true
+thiserror.workspace = true
+tracing.workspace = true
+tokio = { workspace = true, features = ["full"] }
+tokio-stream.workspace = true
+url.workspace = true
+strum.workspace = true
+strum_macros.workspace = true
+parquet.workspace = true
+arrow-array.workspace = true
+fastcrypto = { workspace = true, features = ["copy_key"] }
+mysten-metrics.workspace = true
+sui-analytics-indexer-derive.workspace = true
+sui-indexer.workspace = true
+serde_yaml.workspace = true
+serde_json.workspace = true
+eyre.workspace = true
+rocksdb.workspace = true
```

Comment:
prob some cleanup in the deps from what i see at least rocksdb is not needed

Reply:
Excellent point, I cut this significantly.